Formula/ipfs.rb

```diff
@@ -21,7 +21,7 @@ class Ipfs < Formula
     sha256 cellar: :any_skip_relocation, mojave:   "868371961578f442159865ff5111d778dbc730cda71058f942cbb354e6a46029"
   end
 
-  depends_on "go@1.14" => :build
+  depends_on "go" => :build
 
   def install
     ENV["GOPATH"] = buildpath
```

Comment:
Can you try replacing this:
```patch
-     ENV["GOPATH"] = buildpath
-    (buildpath/"src/github.com/ipfs/go-ipfs").install buildpath.children
-    cd("src/github.com/ipfs/go-ipfs") { system "make", "install" }
-    bin.install "bin/ipfs"
+   system "go", "build", *std_go_args
```

Reply:
Fails with `Errno::ENOENT: No such file or directory @ dir_chdir - src/github.com/ipfs/go-ipfs`.
Official [build documentation](https://github.com/ipfs/go-ipfs#download-and-compile-ipfs) uses `make install` so I think we should keep using it.

Reply:
But GOPATH is deprecated so we should try and get rid of that.

Reply:
>https://github.com/golang/go/wiki/GOPATH: Note that removing GOPATH development mode does not mean removing the GOPATH variable. It will still be used for the purposes listed at the top of this page.

I don't know much about Go, but I think we're just using GOPATH as an environment variable, which isn't being depreciated. IPFS has a go.mod so I think it's not using GOPATH development mode.

Reply:
We're deprecating the usage of GOPATH in homebrew formulae where possible and switching to go modules.

This should work:

```
  def install
    system "make", "build"
    bin.install "cmd/ipfs/ipfs"

    bash_completion.install "misc/completion/ipfs-completion.bash"
  end
```

Could you please add it to the PR.

Reply:
Yep, tested it and it works.